bigbluebutton-tests/playwright/core/helpers.js

```diff
@@ -11,6 +11,24 @@ const { format } = require('node:util');
 const chalk = require('chalk');
 const parameters = require('./parameters');
 
+function getChecksum(text, secret) {
+  let algorithm = (process.env.CHECKSUM || '').toLocaleLowerCase();
+  if (!['sha1', 'sha256', 'sha512'].includes(algorithm)) {
```

Comment:
is there any specific reason / case for using `toLocaleLoweCase()` instead of `toLowerCase()`?

Reply:
It was a typo from auto complete. `toLowerCase()` is fine.